package.json

```diff
@@ -23,22 +23,22 @@
     },
     "devDependencies": {
         "@terascope/eslint-config": "^0.7.1",
-        "@terascope/job-components": "^0.58.5",
-        "@terascope/standard-asset-apis": "^0.6.0",
+        "@terascope/job-components": "^0.59.0",
+        "@terascope/standard-asset-apis": "^0.6.1",
```

Comment:
Why are `job-components` and `standard-asset-apis` devDependencies?  Do you know?

Reply:
Oh, nevermind, this is the top level package.json